tmt/steps/prepare/distgit.py

```diff
@@ -287,11 +287,11 @@ def go(
             self.discover.post_dist_git(created_content)
             # FIXME needs refactor of Prepare, tmt.base etc...
             # doing quick & dirty injection of prepareinstalls
-            for guest in self.step.plan.provision.guests():
+            for g in self.step.plan.provision.guests():
```

Comment:
@martinhoyer was this reported by ruff? I personally rather prefer `guest` here.